testing/src/uploadResult.ts

```diff
@@ -0,0 +1,59 @@
+import axios, { AxiosResponse } from "axios";
+import FormData from "form-data";
+import fs from "fs";
+
+const TQ_USERNAME = process.env.TQ_USER_NAME;
+const TQ_PASSWORD = process.env.TQ_PASSWORD;
+const TQ_SECRET = process.env.TQ_SECRET;
+const TQ_RUN_ID = process.env.TQ_RUN_ID; //( default id is 26606. Will need to change once we have integrated TQ account for the team)
+
+const getAuthToken = async (): Promise<AxiosResponse<any>> => {
+  const url = "https://api.testquality.com/api/oauth/access_token";
```

Comment:
Please move the const `url` in both methods to the top and/or create a const to the base URL that both constants are sharing.